cmd/server/BUILD

```diff
@@ -0,0 +1,16 @@
+package(default_visibility = ["//visibility:public"])
+
+load("@io_bazel_rules_go//go:def.bzl", "go_binary")
+
+go_binary(
+    name = "broker",
```

Comment:
How about we can this brks and then we'll have brkc as a test client? This parallels mixc and mixs.

On a related note, please copy the mixcol command from the mixer repo and use the same pattern in this repo. This is the tool we run in order to generate reference documentation for the command-line tools.